src/services/dumper/templates/agent-nodejs/env.hbs

```diff
@@ -0,0 +1,22 @@
+APPLICATION_PORT={{ port }}
+
+CORS_ORIGINS=
+
+DATABASE_URL={{ databaseUrl }}
+{{#if hasDockerDatabaseUrl}}
+DOCKER_DATABASE_URL={{ dockerDatabaseUrl }}
+{{/if}}
+{{#if dbSchema }}
+DATABASE_SCHEMA={{ dbSchema }}
+{{/if}}
+DATABASE_SSL={{ ssl }}
+# This should be removed in production environment.
+DATABASE_REJECT_UNAUTHORIZED=false
+
+FOREST_AUTH_SECRET={{ forestAuthSecret }}
+FOREST_ENV_SECRET={{ forestEnvSecret }}
+
+{{#if forestUrl}}
+FOREST_URL={{forestUrl}}
+NODE_TLS_REJECT_UNAUTHORIZED=0
+{{/if}}
```

Comment:
```suggestion
FOREST_AUTH_SECRET={{ forestAuthSecret }}
FOREST_ENV_SECRET={{ forestEnvSecret }}
{{#if forestServerUrl}}
FOREST_SERVER_URL: {{ forestServerUrl }}},
NODE_TLS_REJECT_UNAUTHORIZED=0
{{/if}}

APPLICATION_URL={{ applicationUrl }}
APPLICATION_PORT={{ applicationPort }}

DATABASE_URL={{ databaseUrl }}
DATABASE_SSL={{ databaseSsl }}
{{#if dbSchema }}
DATABASE_SCHEMA={{ databaseSchema }}
{{/if}}
```

Reply:
APPLICATION_URL was used before for OIDC (compute the callback the url). It has no use today we can remove it.

FOREST_SERVER_URL differs from the past, we were used to use FOREST_URL. Let's clarify the situation and make a decision